apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.html

```diff
@@ -0,0 +1,186 @@
+                        {{ commonLocaleData?.app_search }}
+                    </button>
+                </div>
+            </div>
+            <div class="mt-5" *ngIf="liabilityPaymentListInProgress$ | async">
+                <giddh-page-loader></giddh-page-loader>
+            </div>
+            <div class="row mt-5" *ngIf="!(liabilityPaymentListInProgress$ | async)">
+                <div class="table-responsive" *ngIf="hasTaxNumber">
+                    <table mat-table [dataSource]="dataSource" class="w-100" *ngIf="dataSource.length">
+                        <ng-container matColumnDef="index">
+                            <th mat-header-cell *matHeaderCellDef>#</th>
+                            <td mat-cell *matCellDef="let element; let i = index">{{ i + 1 }}.</td>
+                        </ng-container>
+                        <ng-container matColumnDef="amount" *ngIf="isPaymentMode">
+                            <th mat-header-cell *matHeaderCellDef class="text-right">
+                                {{ commonLocaleData?.app_amount }}
+                            </th>
+                            <td mat-cell *matCellDef="let element" class="text-right">
+                                <amount-field
+                                    class="d-inline-block"
+                                    *ngIf="element?.amount"
+                                    [amount]="element?.amount"
+                                    [currencySymbol]="activeCompany?.baseCurrencySymbol"
+                                    [currencyCode]="activeCompany?.baseCurrency"
+                                ></amount-field>
+                                <span *ngIf="!element?.amount">-</span>
+                            </td>
+                        </ng-container>
+                        <ng-container matColumnDef="received" *ngIf="isPaymentMode">
+                            <th mat-header-cell *matHeaderCellDef>
+                                {{ localeData?.received_date }}
+                            </th>
+                            <td mat-cell *matCellDef="let element">
+                                {{ element?.received ?? "-" }}
+                            </td>
+                        </ng-container>
+                        <ng-container matColumnDef="from" *ngIf="!isPaymentMode">
+                            <th mat-header-cell *matHeaderCellDef>
+                                {{ commonLocaleData?.app_from }}
+                            </th>
+                            <td mat-cell *matCellDef="let element">
+                                {{ element?.taxPeriod?.from ?? "-" }}
+                            </td>
+                        </ng-container>
+                        <ng-container matColumnDef="to" *ngIf="!isPaymentMode">
+                            <th mat-header-cell *matHeaderCellDef>
+                                {{ commonLocaleData?.app_to | titlecase }}
```

Comment:
what's the use of titlecase pipe here?

Reply:
"app_to": "to",
    
  capitalise To and it will not any impact in hindi and marathi

Reply:
why didn't capitalize from?